site/app/controllers/HomePageController.php

```diff
@@ -101,25 +102,64 @@ public function getGroups($user_id = null): MultiResponse {
      * @return MultiResponse
      */
     public function showHomepage() {
+        $user = $this->core->getUser();
         $courses = $this->getCourses()->json_response->json;
 
+
+
         return new MultiResponse(
             null,
             new WebResponse(
                 ['HomePage'],
                 'showHomePage',
                 $this->core->getUser(),
+                $this->core->getCsrfToken(),
```

Comment:
Is this change necessary?  I believe that every page has access to the CSRF token since it's included in the global header.  (`GlobalHeader.twig`)

Reply:
Not necessary, was removed!